EddiInaraService/InaraAPICmdrProfile.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EddiInaraService
+{
+    public partial class InaraService
+    {
+        // Documentation: https://inara.cz/inara-api-docs/
+
+        // Returns basic information about commander from Inara like ranks, 
+        // squadron, a link to the commander's Inara profile, etc. 
+
+        public InaraCmdr GetCommanderProfile(string commanderName)
+        {
+            return GetCommanderProfiles(new string[] { commanderName } )?.FirstOrDefault();
+        }
+
+        public List<InaraCmdr> GetCommanderProfiles(string[] commanderNames)
```

Comment:
@richardbuckle 	By blocking all sending from the `SendEventBatch` method you are disabling this function while the game is in beta.

Reply:
Yes, I know. The question is whether the value of getting commander profiles in a beta of the game is worth the risk of future development causing accidental sends. I'm leaning towards no but am open to persuasion.

Reply:
I think that users should always be able to get commander data from Inara about a commander regardless of the current game state. I don't think it's reasonable for commanders to expect otherwise.

Reply:
Commanders clearing their game saves or doing other wacky things in beta would be immaterial. 
We wouldn't be sending data that would change a commander's Inara API profile, just retrieving a profile from Inara.

Reply:
Alternate approach for consideration: 
Block all InaraAPIEvents where the eventName does not contain `get`

Reply:
Note that the current approach, coupled with `MakeSafe()`, also completely disables the `SendEventBatch` method for testing.

Reply:
I think this is now resolved?